CoreFoundation/Base.subproj/CFBase.c

```diff
@@ -875,14 +875,16 @@ void _CFRuntimeSetCFMPresent(void *addr) {
 
 
 extern void __HALT() {
-#if defined(__ppc__)
+#if defined(__ppc__) || defined(__arm__)
```

Comment:
actually i think we could probably generalize this into __builtin_trap() since we require clang